Lib/test/test_pdb.py

```diff
@@ -1632,6 +1632,20 @@ def test_module_without_a_main(self):
         self.assertIn("ImportError: No module named t_main.__main__",
                       stdout.splitlines())
 
+    def test_package_without_a_main(self):
+        pkg_name = 't_pkg'
+        module_name = 't_main'
+        os_helper.rmtree(pkg_name)
```

Comment:
Looks like you're missing an `from test.support import os_helper` import for this line to work.

Reply:
Looking at the header, `support` was already imported, so I re-used that import.

Reply:
Ugh. Bad idea. Okay, going with your suggestion ;)

Reply:
Suggestion doesn't work either. I remember now, between 3.9 and 3.10, test.support was refactored.